.evergreen/config.yml

```diff
@@ -2316,6 +2317,7 @@ buildvariants:
         encryption: [ "encryption_crypt_shared" ]
       then:
         remove_tasks:
+          - ".rapid"
```

Comment:
We still want to test rapid versions.

Reply:
Oh interesting, what if we update rapid to r6.1.0-alpha? To do that you create a PR like this: https://github.com/mongodb-labs/drivers-evergreen-tools/pull/217